### PR TITLE
update travis badge and new org addresses

### PR DESCRIPTION
**Pull request by reqshark**
some of #314 fixed but not appveyor; that part still needs fixing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,13 +119,13 @@ This works like a `-I<path-to-NAN>` when compiling your addon.
 <a name="example"></a>
 ## Example
 
-See **[LevelDOWN](https://github.com/rvagg/node-leveldown/pull/48)** for a full example of **NAN** in use.
+See **[LevelDOWN](https://github.com/iojs/node-leveldown/pull/48)** for a full example of **NAN** in use.
```
Comment (kkoopa) [inline README.md]: This has not moved.

Comment (reqshark) [inline README.md]: oops

**Comment by kkoopa** [comment]
Thanks. I assume @rvagg can fix the AppVeyor settings if it requires logging in to the account or such.

**Comment by rvagg** [comment]
I believe that's done on my end now, pointing to iojs/nan

**Pull request merged**